accord-core/src/main/java/accord/messages/BeginRecovery.java

```diff
@@ -78,17 +78,19 @@ public void process(Node node, Id replyToNode, ReplyContext replyContext)
                 // committed txns with an earlier txnid and have our txnid as a dependency
                 earlierCommittedWitness = committedStartedBefore(instance, txnId, txn.keys)
                                           .filter(c -> c.savedDeps().contains(txnId))
-                                          .collect(Dependencies::new, Dependencies::add, Dependencies::addAll);
+                                          .collect(() -> Deps.builder(txn.keys), Deps.Builder::add, (a, b) -> { throw new IllegalStateException(); })
```

Comment:
same logic as line 89, maybe best if we add a `Deps.collector(txn.keys)` that looks like the following

```
public static Collector<Command, Builder, Builder> collector(Keys keys)
{
    return Collector.of(() -> Deps.builder(keys), Deps.Builder::add, (a, b) -> { throw new IllegalStateException(); });
}
```